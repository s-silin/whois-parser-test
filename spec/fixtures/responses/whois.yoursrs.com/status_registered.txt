Domain Name: yoursrs.com
Registry Domain ID: 107171410_domain_com-vrsn
Registrar WHOIS Server: whois.yoursrs.com
Registrar URL: http://www.realtimeregister.com
Updated Date: 2017-11-26T00:43:08Z
Creation Date: 2003-11-24T09:40:55Z
Registrar Registration Expiration Date: 2018-11-24T09:40:55Z
Registrar: REALTIME REGISTER B.V.
Registrar IANA ID: 839
Reseller: Realtime Register B.V.
Domain Status: ok http://www.icann.org/epp#ok
Registry Registrant ID: REDACTED FOR PRIVACY
Registrant Name: REDACTED FOR PRIVACY
Registrant Organization: REDACTED FOR PRIVACY
Registrant Street: REDACTED FOR PRIVACY
Registrant City: REDACTED FOR PRIVACY
Registrant State/Province: 
Registrant Postal Code: REDACTED FOR PRIVACY
Registrant Country: NL
Registrant Phone: REDACTED FOR PRIVACY
Registrant Phone Ext: REDACTED FOR PRIVACY
Registrant Fax: REDACTED FOR PRIVACY
Registrant Fax Ext: REDACTED FOR PRIVACY
Registrant Email: https://mydomainprovider.com/contact_domain/
Registry Admin ID: REDACTED FOR PRIVACY
Admin Name: REDACTED FOR PRIVACY
Admin Organization: REDACTED FOR PRIVACY
Admin Street: REDACTED FOR PRIVACY
Admin City: REDACTED FOR PRIVACY
Admin State/Province: REDACTED FOR PRIVACY
Admin Postal Code: REDACTED FOR PRIVACY
Admin Country: REDACTED FOR PRIVACY
Admin Phone: REDACTED FOR PRIVACY
Admin Phone Ext: REDACTED FOR PRIVACY
Admin Fax: REDACTED FOR PRIVACY
Admin Fax Ext: REDACTED FOR PRIVACY
Admin Email: https://mydomainprovider.com/contact_domain/
Registry Tech ID: REDACTED FOR PRIVACY
Tech Name: REDACTED FOR PRIVACY
Tech Organization: REDACTED FOR PRIVACY
Tech Street: REDACTED FOR PRIVACY
Tech City: REDACTED FOR PRIVACY
Tech State/Province: REDACTED FOR PRIVACY
Tech Postal Code: REDACTED FOR PRIVACY
Tech Country: REDACTED FOR PRIVACY
Tech Phone: REDACTED FOR PRIVACY
Tech Phone Ext: REDACTED FOR PRIVACY
Tech Fax: REDACTED FOR PRIVACY
Tech Fax Ext: REDACTED FOR PRIVACY
Tech Email: https://mydomainprovider.com/contact_domain/
Name Server: ns1.yoursrs.com
Name Server: ns2.yoursrs.com
DNSSEC: not signed
Registrar Abuse Contact Email: abuse[at]realtimeregister.com
Registrar Abuse Contact Phone: +31.384530759
URL of the ICANN WHOIS Data Problem Reporting System:
http://wdprs.internic.net/
>>> Last update of WHOIS database: 2018-08-27T19:28:08Z <<<

For more information on Whois status codes, please visit https://icann.org/epp

Please email the listed admin email address if you wish to raise a legal issue.

The data in this whois database is provided to you for information purposes only, that is,
to assist you in obtaining information about or related to a domain name registration record.
We make this information available "as is," and do not guarantee its accuracy.
By submitting a whois query, you agree that you will use this data only for lawful purposes and that, under no circumstances will you use this data to:
(1) enable high volume, automated, electronic processes that stress or load this whois database system providing you this information; or
(2) allow, enable, or otherwise support the transmission of mass unsolicited, commercial advertising or solicitations via direct mail,
electronic mail, or by telephone. The compilation, repackaging, dissemination or other use of this data is expressly prohibited without
prior written consent from REALTIME REGISTER B.V.

The domain listed is registered under privacy protect conditions.
To contact the registrant, please e-mail the Registrant Email.

