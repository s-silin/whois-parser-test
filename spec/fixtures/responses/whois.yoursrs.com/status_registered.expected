#domain
  %s == "yoursrs.com"

#domain_id
  %s == "107171410_domain_com-vrsn"


#status
  %s == :registered

#available?
  %s == false

#registered?
  %s == true


#created_on
  %s %CLASS{time}
  %s %TIME{2003-11-24 09:40:55Z}

#updated_on
  %s %CLASS{time}
  %s %TIME{2017-11-26T00:43:08Z}

#expires_on
  %s %CLASS{time}
  %s %TIME{2018-11-24T09:40:55Z}


#registrar
  %s %CLASS{registrar}
  %s.id           == '839'
  %s.name         == "REALTIME REGISTER B.V."
  %s.organization == "REALTIME REGISTER B.V."
  %s.url          == "http://www.realtimeregister.com"


#registrant_contacts
  %s %CLASS{array}
  %s %SIZE{1}
  %s[0] %CLASS{contact}
  %s[0].type         == Whois::Parser::Contact::TYPE_REGISTRANT

#admin_contacts
  %s %CLASS{array}
  %s %SIZE{1}
  %s[0] %CLASS{contact}
  %s[0].type         == Whois::Parser::Contact::TYPE_ADMINISTRATIVE

#technical_contacts
  %s %CLASS{array}
  %s %SIZE{1}
  %s[0] %CLASS{contact}
  %s[0].type         == Whois::Parser::Contact::TYPE_TECHNICAL

#nameservers
  %s %CLASS{array}
  %s %SIZE{2}
  %s[0] %CLASS{nameserver}
  %s[0].name == "ns1.yoursrs.com"
  %s[1] %CLASS{nameserver}
  %s[1].name == "ns2.yoursrs.com"
